moto/guardduty/urls.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
+from .responses import GuardDutyResponse
+
+response = GuardDutyResponse()
+
+url_bases = [
+    "https?://guardduty.(.+).amazonaws.com",
+]
+
+
+url_paths = {
+    "{0}/detector$": response.detector,
```

Comment:
If you want to extend this service with more methods, each method would have to be listed here.
(APIGateway is a good example on what this would look like: https://github.com/spulec/moto/blob/master/moto/apigateway/urls.py)